llvm/include/llvm/IR/IntrinsicsDirectX.td

```diff
@@ -72,7 +72,7 @@ def int_dx_udot :
     [llvm_anyint_ty, LLVMScalarOrSameVectorWidth<0, LLVMVectorElementType<0>>],
     [IntrNoMem, Commutative] >;
 
-def int_dx_frac  : DefaultAttrsIntrinsic<[llvm_anyfloat_ty], [LLVMMatchType<0>]>;
+def int_dx_frac  : DefaultAttrsIntrinsic<[llvm_anyfloat_ty], [LLVMMatchType<0>], [IntrNoMem]>;
```

Comment:
These changes should be split out into a separate PR.

Reply:
https://github.com/llvm/llvm-project/pull/108852